src/components/templates/appProvider.tsx

```diff
@@ -59,15 +90,31 @@ export const AppProvider = ({ children }: AppProviderProps) => {
 		if (configStep && !shouldShowStepOnPath(configStep, location.pathname) && !!tourProjectExists) {
 			openModal(ModalName.continueTour, { name: tours?.[activeTour?.tourId]?.name });
 		}
+
 		// eslint-disable-next-line react-hooks/exhaustive-deps
 	}, []);
 
+	const onContactSupportClick = () => {
+		closeModal(ModalName.quotaLimit);
+		try {
+			window.open(`mailto:${supportEmail}`, "_blank");
+			// eslint-disable-next-line @typescript-eslint/no-unused-vars
+		} catch (error) {
+			const mailtoLink = document.createElement("a");
+			mailtoLink.href = "mailto:haim@autokitteh.com";
```

Comment:
"haim@autokitteh.com" add to const file or use `supportEmail`
```suggestion
			mailtoLink.href = "mailto:haim@autokitteh.com";
```

Reply:
added to global const.